src/http_server.cc

```diff
@@ -2538,6 +2550,8 @@ HTTPAPIServer::EVBufferToInput(
           v_idx++;
         }
 
+        LittleEndianToHost(dtype, base, base_size, partial_result, next_offset);
```

Comment:
Does this not complain that `partial_result` isn't defined when `TRITON_BIG_ENDIAN` is undefined?

Reply:
get's eaten up